documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,50 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+* Setting parameter `invalidateCache` to `true`, will clean up the cache of a Fiori Launchpad site, refreshing the content of any applications deployed there. This is **applicable** only when the application is deployed and accessed via Portal service (Fiori Launchpad site).
+
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+### OAuth credential creation
+
+  1 Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
```

Comment:
Need to add a dot so the md parser understands it's a numbered list. Indentation is then not necessary.
```suggestion
1. Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
```

Reply:
Use OAuth as this is the official capitalisation